tests/playwright/README.md

```diff
@@ -1,146 +1,103 @@
 
-Since you have your tests and testing framework at place, you can now run your tests from the repository.
+- Under `devDependencies`:
+  - `"@podman-desktop/tests-playwright": "next",`[²]
+  - `"@playwright/test": "^1.48.1"` (or the current latest)
+- Under `scripts`:
+  - `"test:e2e:setup": "xvfb-maybe --auto-servernum --server-args='-screen 0 1280x960x24' --"` [³]
+  - `"test:e2e": "cross-env E2E_TESTS=true npm run test:e2e:setup npx playwright test tests/src"`
 
-You will have to checkout podman-desktop repository and build it first.
+8. Execute `pnpm install`, which should extract the contents of the previously built Podman Desktop into the `node_modules` folder in your repo
+9. Write your E2E tests on your repo, which may use your changes to `@podman-desktop/tests-playwright` from step 3 (optional)
+10. Run your E2E tests by executing `pnpm test:e2e`
 
-1. `git clone https://github.com/containers/podman-desktop`
-2. `cd podman-desktop`
-3. `pnpm install`
-4. `pnpm test:e2e:build` -> this step is essential
+[¹] Remember that environment variables defined this way only work on the terminal they were defined and only for as long as the terminal is active.
 
-Then you need to prepare your tests to be run from your repository
+[²] Using the value "next" works for local executions of the tests, but if you want to execute the tests remotely you should specify the latest version of the framework, you can find that value [here](https://www.npmjs.com/package/@podman-desktop/tests-playwright).
 
-0. Add dependency for `@podman-desktop/tests-playwright` in `devDependencies`
-1. add npm script target to run E2E tests:
+[³] If your project does not already have the `xvfb-maybe` dependency, you'll need to add it as well
 
-```package.json
-  "scripts": {
-    "test:e2e": "PODMAN_DESKTOP_ARGS='/path/to/podman-desktop' vitest run tests/src/ --pool=threads --poolOptions.threads.singleThread --poolOptions.threads.isolate --no-file-parallelism",
-  }
-```
+## How to develop using locally built @podman-desktop/tests-playwright
 
-2. Implement your E2E tests in `tests` folder of YOUR repo
-3. Run `npm test:e2e`
-4. Artifacts logs are available under `./tests/output`
+This section references how to use @podman-desktop/tests-playwright generated archive file for local development
+
+1. Get into the Podman Desktop repo folder (should be named `podman-desktop`)
+2. Install its local dependencies by executing `pnpm install`
+3. Implement your changes to the E2E tests library (it can be found under `tests/playwright`)
+4. Build the application with `pnpm build`
+5. Create the local package with `pnpm run package`, this will produce a .tgz archive
+6. In YOUR repository, update the `package.json` file to have the following contents under `devDependencies`:
+   `"@podman-desktop/tests-playwright": "file:../podman-desktop/tests/playwright/podman-desktop-tests-playwright-1.9.0.tgz"`
```

Comment:
version is derived from package.json from `tests/playwright`. nice to have update, no need for it.